legal.markdown

```diff
@@ -62,6 +62,7 @@ CFEngine includes the following 3rd party libraries and components:
 * [redis](http://redis.io) under the [three clause BSD license](http://redis.io/topics/license)
 * [rsync](http://rsync.samba.org) under the [GPLv3](http://rsync.samba.org/GPL.html)
 * [Twitter Bootstrap Framework](http://getbootstrap.com) under [Apache 2.0](http://www.apache.org/licenses/LICENSE-2.0)
+* [Bootstrap Icons](https://icons.getbootstrap.com) under the [MIT license](https://github.com/twbs/icons/blob/main/LICENSE.md)
```

Comment:
There should not be a space between `[MIT license]` and `(https://github.com/twbs/icons/blob/main/LICENSE.md)`

Reply:
Invalid markdown, won't create a link if there is a space.

Reply:
My mistake, it's fine! line wrapping got me.